2023-07-03 16:13:30,044: [11] m.main:INFO: Registered plugins: IntrospectionAPIPlugin, <class 'manticore.ethereum.plugins.SkipRevertBasicBlocks'>, <class 'manticore.ethereum.plugins.FilterFunctions'>
2023-07-03 16:13:30,045: [11] m.main:INFO: Beginning analysis
2023-07-03 16:13:30,047: [11] m.e.manticore:INFO: Starting symbolic create contract
2023-07-03 16:13:30,840: [11] m.e.manticore:INFO: Starting symbolic transaction: 0
2023-07-03 16:13:30,977: [11] m.e.manticore:INFO: 0 alive states, 1 terminated states
2023-07-03 16:13:31,039: [11] m.e.manticore:INFO: Starting symbolic transaction: 1
2023-07-03 16:13:31,245: [11] m.c.manticore:INFO: Results in /mcore_dlx23i6w
2023-07-03 16:13:31,245: [11] m.c.manticore:INFO: Total time: 0.3983795642852783
2023-07-03 16:13:31,943: [70] m.main:INFO: Registered plugins: IntrospectionAPIPlugin, <class 'manticore.ethereum.plugins.SkipRevertBasicBlocks'>, <class 'manticore.ethereum.plugins.FilterFunctions'>
2023-07-03 16:13:31,943: [70] m.main:INFO: Beginning analysis
2023-07-03 16:13:31,945: [70] m.e.manticore:INFO: Starting symbolic create contract
2023-07-03 16:13:35,101: [70] m.e.manticore:INFO: Found a concrete globalsha3 b'EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)' -> 63076024560530113402979550242307453568063438748328787417531900361828837441551
2023-07-03 16:13:37,060: [70] m.e.manticore:INFO: Found a concrete globalsha3 b'Arthswap LPs' -> 19089632945547805436458783591964997020421063286102968747771341591983408473891
2023-07-03 16:13:40,780: [70] m.e.manticore:INFO: Found a concrete globalsha3 b'1' -> 90743482286830539503240959006302832933333810038750515972785732718729991261126
2023-07-03 16:13:51,785: [70] m.e.manticore:INFO: Found a concrete globalsha3 b'\x8bs\xc3\xc6\x9b\xb8\xfe=Q.\xccL\xf7Y\xccy#\x9f{\x17\x9b\x0f\xfa\xca\xa9\xa7]R+9@\x0f*4Yo\\#*%\x97<dy>\xbc\xa2\xdf\xcc\x11\x03\x89\xe9\x84\xf1C\xe8\xa8\xdc\xd1\xd4:\xe7#\xc8\x9e\xfd\xaaT\xc0\xf2\x0cz\xdfa(\x82\xdf\tP\xf5\xa9Qc~\x03\x07\xcd\xcbLg/)\x8b\x8b\xc6\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\xcb7\xf40M\xe8\xd1\xca-\xf2GLn:\x00\xab>r\x04b' -> 65460150144571707239382082296729992324486228279372715776196172787884391044412
2023-07-03 16:13:53,335: [70] m.e.manticore:INFO: Starting symbolic transaction: 0
2023-07-03 16:15:28,750: [70] m.e.manticore:INFO: Found a concrete globalsha3 b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02' -> 103897324152558924508739690356745203242964415294502778034921468049474951696033
2023-07-03 16:16:11,644: [70] m.e.manticore:INFO: Found a concrete globalsha3 b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x02\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01' -> 27376155845717860833342594873357525089589924219251518322086369159840883582034
2023-07-03 16:16:13,530: [70] m.c.worker:ERROR: Exception in state 2: ManticoreError('Forking on unfeasible constraint set')
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/manticore/core/worker.py", line 137, in run
    current_state.execute()
  File "/usr/local/lib/python3.7/dist-packages/manticore/ethereum/state.py", line 8, in execute
    return self._platform.execute()
  File "/usr/local/lib/python3.7/dist-packages/manticore/platforms/evm.py", line 3106, in execute
    self.current_vm.execute()
  File "/usr/local/lib/python3.7/dist-packages/manticore/platforms/evm.py", line 1322, in execute
    raise Concretize("Symbolic PC", expression=expression, setstate=setstate, policy="ALL")
manticore.core.state.Concretize

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/manticore/core/worker.py", line 158, in run
    m._fork(current_state, exc.expression, exc.policy, exc.setstate, exc.values)
  File "/usr/local/lib/python3.7/dist-packages/manticore/core/manticore.py", line 518, in _fork
    raise ManticoreError("Forking on unfeasible constraint set")
manticore.exceptions.ManticoreError: Forking on unfeasible constraint set
 
